cdap-security/src/main/java/co/cask/cdap/security/securestore/FileSecureStoreData.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.security.securestore;
+
+import co.cask.cdap.api.security.securestore.SecureStoreData;
+import co.cask.cdap.api.security.securestore.SecureStoreMetadata;
+
+/**
+ * Represents the data stored in the Secure Store.
+ */
+public class FileSecureStoreData implements SecureStoreData {
```

Comment:
This implementation has nothing to do with File. I think `SecureStoreData` doesn't need to be an interface, but rather just a POJO.